site/content/api/Credentials-API.md

```diff
@@ -320,13 +320,19 @@ The next example contains two [pre-shared secrets]({{< relref "#pre-shared-key"
 
 Protocol Adapters are responsible for authenticating devices when they connect. The Credentials API provides the [Get Credentials]({{< relref "#get-credentials" >}}) operation to support Protocol Adapters in doing so as illustrated below:
 
-**TBD sequence diagram**
+The following sequence diagram illustrates the flow of messages involved in a *Protocol Adapter* authenticating a device that provides it's authentication data.
```

Comment:
IMHO it would be much easier to use a concrete authentication method as an example, e.g. MQTT username/password. This would allow us to use a *CONNECT* packet and *hashed-password* as type in the sequence diagram. I guess the reader will then be able to apply the pattern to a different authentication mechanism.